R/g.loadlog.R

```diff
@@ -7,6 +7,7 @@ g.loadlog = function(loglocation = c(), coln1 = c(), colid = c(), nnights = c(),
   # Load sleep log data...
   S = data.table::fread(loglocation, stringsAsFactors = FALSE, data.table = FALSE,
                         check.names = TRUE, colClasses = "character")
+  nnights = floor((ncol(S) - 1) / 2)
```

Comment:
Replace by:
`nnights = floor((ncol(S) - coln1 + 1)) / 2)`

because your implementation assumes that data starts on second column.